EKS_DISTRO_TAG_FILE.yaml

```diff
@@ -3,7 +3,7 @@ al2:
   eks-distro-minimal-base: 2023-09-06-1694026927.2
   eks-distro-minimal-base-nonroot: 2023-09-06-1694026927.2
   eks-distro-minimal-base-glibc: 2023-09-13-1694631680.2
-  eks-distro-minimal-base-iptables: 2023-09-13-1694631680.2
+  eks-distro-minimal-base-iptables: null
```

Comment:
why is this set to null now?

Reply:
Setting these values to null trigger automation to force rebuild the images. https://github.com/aws/eks-distro-build-tooling/pull/1166